sunpy/net/dataretriever/client.py

```diff
@@ -188,6 +188,10 @@ def _makeargs(self, *args, **kwargs):
         \*\*kwargs: `dict`
             None.
         """
+#       Currently Fido doesn't support Parsing Wavelength support.
+#       i.e. If you pass Wavelength object it will ot recognize it.
+#       To do recognize it, we store a key called wavelength and make it
+#       hold a .vso.attrs.Wavelength object.
```

Comment:
I think this block doesn't apply anymore... probably is from one of the rebases...